examples/models/metadata/example-grpc.yaml

```diff
@@ -0,0 +1,58 @@
+
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: graph-metadata-grpc
+spec:
+  name: test-deployment
+  predictors:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: seldonio/metadata-generic-node:0.3
+          name: node-one
+          env:
+          - name: API_TYPE
+            value: GRPC             
+          - name: MODEL_METADATA
+            value: |
+              ---
+              name: node-one
+              versions: [ generic-node/v0.3 ]
+              platform: seldon
+              inputs:
+              - messagetype: tensor
+                schema:
+                  names: [one-input]
+              outputs:
+              - messagetype: tensor
+                schema:
+                  names: [one-output]
+        - image: seldonio/metadata-generic-node:0.3
+          name: node-two
+          env:
+          - name: API_TYPE
```

Comment:
This should not be passed by env var, it should be passed via the graph type

Reply:
Does graph type sets environmental variable on the container? 
I wanted to use the same image without rebuilding it.

Not sure if graph-level var is used anymore...